dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteThread.java

```diff
@@ -460,10 +474,12 @@ private boolean needComplementProcess() {
     }
 
     private void startProcess() throws Exception {
-        buildFlowDag();
         if (this.taskInstanceHashMap.size() == 0) {
+            isStart = false;
```

Comment:
The assignment here is not need

Reply:
In the complement scenario,   the 'start process' will run multiple times.